app/Config/core.php

```diff
@@ -176,7 +176,7 @@
  * for instance. Each version can then have its own view cache namespace.
  * Note: The final cache file name will then be `prefix_cachefilename`.
  */
-	//Configure::write('Cache.viewPrefix', 'prefix');
+    //Configure::write('Cache.viewPrefix', 'prefix');
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('Cache.viewPrefix', 'prefix');" but found "//Configure::write('Cache.viewPrefix', 'prefix');"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment